### PR TITLE
Tara-Weather App

### PR DESCRIPTION
**Pull request by trangtara**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        output += `<p class="day"> ${weekday}<span class="icon-image"> ${iconImg}</span> <span class="temper">${temperature}°C</span></p>`;
+      });
+
+      document.getElementById('forecast').innerHTML = output;
+    });
+}
+
+function updateWeatherData() {
+  const todayWeatherUrl = `https://api.openweathermap.org/data/2.5/weather?q=${selectedCityName}&units=metric&APPID=${API_KEY}`
+  const fiveDaysForecastUrl = `https://api.openweathermap.org/data/2.5/forecast?q=${selectedCityName}&units=metric&appid=${API_KEY}`
```
Comment (Andrea-Osmar) [inline code/script.js:112]: indentation here :)

**Review by Andrea-Osmar — APPROVED** [review]
Tara you have done an amazing job!! I am really impressed and I learned so much from going thru your code. Great job with naming functions and keeping alla your code clean and dry. I wouldnt mind some comments in your code to explain what different sections are for. I think the design is great and the overall layout looks good on both desktop and iphone.  I really didnt have much to give comments on and I give you a big golden star for this project!!  🥇

File: code/assets/thunderstorm-sun.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...-6.9 15.1-9.9 1.5-8.4 3.9-16.5 6.8-24.3-14.3-25.7-41.5-43.4-73-43.4-46.2 0-83.7 37.6-83.7 83.8s37.5 83.8 83.7 83.8c.3 0 .6-.1.9-.1 1.1-11.4 3.7-22.4 7.7-32.8-2.8.5-5.6.9-8.5.9-28.5 0-51.7-23.2-51.7-51.7-.1-28.6 23.1-51.9 51.6-51.9zm336 164h-59.5l11-44.1c1.2-4.8.1-9.8-2.9-13.7-3-3.9-7.7-6.2-12.6-6.2h-80c-7.3 0-13.8 5-15.5 12.1l-32 128C334 390.3 341.7 400 352 400h61.6l-13.4 93.7c-1.6 11.1 7.7 18.2 15.8 18.2 8.5 0 12.7-6.2 13.6-7.6 3.4-5.3 111.9-175.8 111.9-175.8 6.8-10.7-1-24.5-13.5-24.5zm-85.7 121.2l5.6-38.9c.7-4.6-.7-9.2-3.8-12.7-3-3.5-7.5-5.5-12.1-5.5h-59.5l24-96h47l-11 44.1c-2.5 10.2 5.2 19.9 15.5 19.9h50.8c-24.1 37.9-42.6 66.9-56.5 89.1zm133-227.9C570 158.2 536.5 128 496 128c-8.6 0-17 1.4-25.2 4.3-19.7-23-48.2-36.3-78.8-36.3-56.5 0-102.7 45.3-104 101.6-37.8 13.3-64 49.3-64 90.4 0 47.5 34.8 86.8 80.1 94.4.1-3.4.5-6.7 1.3-10.1l5.3-21.3c-30.9-4.5-54.8-30.9-54.8-63.1 0-30.6 21.8-57 52-62.8l14.5-2.8-2-18c-.2-1.5-.4-2.9-.4-4.4 0-39.7 32.3-72 72-72 24.3 0 46.8 12.2 60.2 32.7l8.1 12.4 13-7.1c32.7-17.8 70.7 8.2 70.8 40.4l-.2 16.2 12.8 2.6c29.8 6 51.3 32.3 51.3 62.7 0 27-16.9 50-40.5 59.4-2.8 4.3-11.6 18.2-23.3 36.6 52.9-.1 95.8-43.1 95.8-96 0-41.1-26.6-77.4-64.7-90.5zM560 208.8z"/></svg>
```
[the remark this answers is not in the document]
Comment (HIPPIEKICK) [reply code/assets/thunderstorm-sun.svg:1]: That's just how the image format SVG looks like 😅 No need to understand it to use it - but if you're interested: https://www.w3schools.com/graphics/svg_intro.asp

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,223 @@
+const API_KEY = 'e27fc7790a6a4c3537de471b9d7612ce'
```
Comment (HIPPIEKICK) [inline code/script.js:1]: ⭐

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+  }
+]
+
+const selectElement = document.getElementById('city-select');
+CITIES.forEach((item) => {
+  selectElement.innerHTML += `<option value="${item.name}">${item.name}</option>`
+})
+
+//get location of the user-ask for consent
+function showCurrentLocation() {
```
Comment (HIPPIEKICK) [inline code/script.js:57]: `const showCurrentLocation = () => {`

File: code/script.js [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+      poster: './assets/dancer-2793110_1920.jpg',
+      message: `Enjoy different kinds of weather in ${cityName} today`
+    }
+  ]
+  return weatherTemplate.find((item) => item.id === weatherId);
+}
+
+
+
+
```
Comment (HIPPIEKICK) [inline code/script.js:223]: Unnecessary linebreaks

File: code/script.js [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+  }, 5000)
+  document.querySelector('.background-container').innerHTML = `
+    <video class="background-video" autoplay muted loop
+      poster= ${weatherTemplate.poster} >
+      <source src="${weatherTemplate.videoSrc}" type="video/mp4">
+    </video>
+  `;
+  document.getElementById('sumMessage').innerHTML = weatherTemplate.message;
+  document.querySelector('.main-container').style.background = weatherTemplate.background;
+  document.querySelector('.main-container').style.color = weatherTemplate.color;
```
Comment (HIPPIEKICK) [inline code/script.js:158]: Try to keep the styling to the CSS-file

**Review by HIPPIEKICK — APPROVED** [review]
Great job Tara! 👍 Some missing semicolons - and a question - why not arrow functions this time? :)

**Pull request closed without merging**